domains/retr0xraiden.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Raiden-16F7",
+        "email": "fahad.rizwan07@gmail.com"
+    },
+    "record": {
+        "URL": "https://raiden-16f7.github.io/portfolio/"
```

Comment:
404 GitHub page.

Reply:
I pasted the URL as is using dashboard, but it changed it to all lowercase.